packages/mock-server/src/data/data.ts

```diff
@@ -767,3 +768,5 @@ export let RequisitionLineData = [
 
 export let StocktakeData = createStocktakes();
 export let StocktakeLineData = createStocktakeLines();
+
+("I missed the convo cause I usually have channel on mute - ya'll just be speaking some bullshit all day");
```

Comment:
true. but needed here?

Reply:
Hmm. I thought people needed to know. I guess i'll remove it!

<img src="https://media3.giphy.com/media/jUwpNzg9IcyrK/giphy.gif?cid=5a38a5a2zplf8o6syy4ibka4jst3labjoe6c6wvm2w750w5f&rid=giphy.gif&ct=g"/>